src/pages/homepage.tsx

```diff
@@ -1,482 +1,212 @@
-            )}
-            desc={t('homepage:showcase.buttonblock.desc', 'Let the community know about it here')}
-            fillParent={true}
-          >
-            <Link
-              href={t('homepage:showcase.buttonblock.href', 'mailto:contact@opentermsarchive.org')}
-            >
-              <a title={t('homepage:showcase.buttonblock.link.title', 'Send us a mail')}>
-                <Button>{t('homepage:showcase.buttonblock.label', 'Contact us')}</Button>
-              </a>
-            </Link>
-          </ButtonBlock>
-        </Column>
+            </ButtonBlock>
+          </ButtonBlockList>
+        </Container>
       </Container>
-
-      {/* Partners */}
-      <Container layout="fluid" gridCols="12" gridGutters="11" flex={true} paddingX={false}>
-        <ThumbGalery
-          title={t('homepage:partners.title', 'Our Partners')}
-          subtitle={t('homepage:partners.subtitle', 'They make Open Terms Archive existing')}
-          titleLevel="h3"
-        >
-          <Link href="https://disinfo.quaidorsay.fr">
-            <a target="_blank">
-              <ThumbGaleryItem src="/images/logo-ambnum.png" width="158" height="80" />
-            </a>
-          </Link>
-          <Link href="https://disinformationindex.org/">
-            <a target="_blank">
-              <ThumbGaleryItem src="/images/logo-gdi.png" width="150" height="32" />
-            </a>
-          </Link>
-          <ThumbGaleryItem src="/images/nextgen.png" width="191" height="48" />
-          <ThumbGaleryItem src="/images/france-relance.png" width="77" height="77" />
-          <Link href="https://www.peren.gouv.fr/">
-            <a target="_blank">
-              <ThumbGaleryItem src="/images/logo-peren.png" width="150" height="121" />
-            </a>
-          </Link>
-        </ThumbGalery>
+      {/* Reuses */}
+      <Container gridCols="12" gridGutters="11">
+        <CardList title={t('homepage:reuses.title')} subtitle={t('homepage:reuses.subtitle')}>
+          <Card
+            image="/images/reuses/tosdr.jpg"
+            title="Terms of Service; Didn’t Read"
+            subtitle="Permet aux utilisateurs de donner collaborativement une note aux conditions d’utilisation selon le respect de leurs droits."
+            author="ToS;DR"
+            link="https://tosdr.org"
+            center={true}
+          ></Card>
+          <Card
+            image="/images/reuses/disinfo-experiments.jpg"
+            title={t('homepage:reuses.card2.title')}
+            subtitle={t('homepage:reuses.card2.subtitle')}
+            author={t('homepage:reuses.card2.author')}
+            link="https://disinfo.quaidorsay.fr/en/open-terms-archive/experiments"
+            center={true}
+          ></Card>
+          <Card
+            image="/images/reuses/scripta-manent.jpg"
+            title={t('homepage:reuses.card1.title')}
+            subtitle={t('homepage:reuses.card1.subtitle')}
+            author={t('homepage:reuses.card1.author')}
```

Comment:
Please use semantic names to support translation and maintenance, with same names between image file and translation keys 🙂 
```suggestion
            author={t('homepage:reuses.scripta-manent.author')}
```